### PR TITLE
Upgrade/Backup updates per Ale

### PR DESCRIPTION
**Pull request by paigehargrave**

### CODE REVIEW

File: modules/ROOT/pages/backup-and-disaster-recovery.adoc [hunk 1]
```diff
@@ -27,6 +27,22 @@ Specifies the deadline for the backup job, for example, 30s or 5m. Default is 20
...
+
+[%header%autowidth.spread]
+|===
+| Information | Included in PCE Backup 
+| Databases | Y
+| Configurations | Y
+| Secrets | Y
+| Object Store | Y
+| Drives | N
+| NFS Server Configuration | N
```
Comment (alejandrorascovan) [inline modules/ROOT/pages/backup-and-disaster-recovery.adoc]: It's not that server configuration that is not included, but the NFS files

File: modules/ROOT/pages/backup-and-disaster-recovery.adoc [hunk 2]
```diff
@@ -47,14 +63,21 @@ scp /backup-path/to-restore.tar.gz your_username@remotehost.edu:/var/lib/gravity
...
 * `--log-file`: outputs logs to a file
+====
++
+Wait until the operation completes. This operation takes a few minutes. 
 
-Note that this operation takes a few minutes. 
+. Manually restore the following information: 
++
+* Runtimes
+* NFS server configuration
```
Comment (alejandrorascovan) [inline modules/ROOT/pages/backup-and-disaster-recovery.adoc]: Same as before

File: modules/ROOT/pages/backup-and-disaster-recovery.adoc [hunk 3]
```diff
@@ -47,14 +63,21 @@ scp /backup-path/to-restore.tar.gz your_username@remotehost.edu:/var/lib/gravity
...
 * `--timeout`: specifies the deadline for the backup job (e.g.: 30s, 5m, etc). Default is 20m.
 * `--log-file`: outputs logs to a file
+====
++
+Wait until the operation completes. This operation takes a few minutes. 
 
-Note that this operation takes a few minutes. 
+. Manually restore the following information: 
++
+* Runtimes
```
Comment (alejandrorascovan) [inline modules/ROOT/pages/backup-and-disaster-recovery.adoc]: You don't need to manually restore runtimes

File: modules/ROOT/pages/upgrade.adoc [hunk 4]
```diff
@@ -4,24 +4,28 @@ include::_attributes.adoc[]
 endif::[]
 
 == Before You Begin
+Because Anypoint Platform Private Cloud Edition (Anypoint Platform PCE) upgrades require a working installation, all environment prerequisites should already be met. As long as you (or your infrastructure team) successfully performed all prerequisite tasks listed in xref:prereq-hardware.adoc[Hardware Prerequisites], xref:prereq-software.adoc[Software Prerequisites], and xref:prereq-network.adoc[Network Prerequisites] prior to the installation of Anypoint Platform PCE, your environment can be upgraded successfully.
```
Comment (alejandrorascovan) [inline modules/ROOT/pages/upgrade.adoc:7]: For 1.7.x -> 2.0.x an NFS is needed

File: modules/ROOT/pages/upgrade.adoc [hunk 5]
```diff
@@ -4,24 +4,28 @@ include::_attributes.adoc[]
...
 
-Refer to the xref:release-notes/pce/anypoint-private-cloud-2.0.3-release-notes#upgrade-paths[Upgrade Paths) section in the Anypoint Platform Private Cloud Edition (Anypoint Platform PCE) release notes for information about upgrading Anypoint Platform PCE to the most recent version.
+Refer to the xref:release-notes::pce/anypoint-private-cloud-2.0.3-release-notes#upgrade-paths[Upgrade Paths) section in the Anypoint Platform PCE release notes for information about upgrading Anypoint Platform PCE to the most recent version.
 
 See xref:1.6@upgrade.adoc[About Migrating Anypoint Platform Private Cloud] for information on upgrading to version 1.6.1. See xref:1.7@upgrade.adoc[About Migrating Anypoint Platform Private Cloud] for information on upgrading to version 1.7.1.
 
-[WARNING]
-====
-Before upgrading:
+If you are using AWS, obtain the keys used to run the AWS provisioner. These are required for access EC2, VPC, and S3.
```
Comment (alejandrorascovan) [inline modules/ROOT/pages/upgrade.adoc]: This is only for installations. It's not necessary to run the AWS provisioner for updates.

File: modules/ROOT/pages/upgrade.adoc [hunk 6]
```diff
@@ -4,24 +4,28 @@ include::_attributes.adoc[]
...
 
-The following steps describe how to migrate Anypoint Platform PCE and the Anypoint Platform PCE infrastructure. You must also upgrade your applications to work with the new version of Anypoint Platform PCE. Contact your MuleSoft customer success representative for more information.
+The following steps describe how to upgrade Anypoint Platform PCE and the Anypoint Platform PCE infrastructure. 
 
-== Procedure
+[NOTE]
+====
+* During an upgrade, all Anypoint Platform PCE components are updated.
+* The upgrade process causes downtime for each node as it is updated.
+* The upgrade process does not include your applications and data. You must upgrade your applications to work with the new version of Anypoint Platform PCE.
```
Comment (alejandrorascovan) [inline modules/ROOT/pages/upgrade.adoc]: Not sure what you are trying to say here. Runtime applications are independant from PCE upgrades.

File: modules/ROOT/pages/upgrade.adoc [hunk 7]
```diff
@@ -90,6 +94,13 @@ Before accessing Anypoint Platform PCE, ensure that you configure your NSF serve
 .. Select *Save*.
 .. Select *Test NFS Server* to test that your NFS server is configured correctly.
 
+. Unless you are updating Anypoint Platform PCE from 2.0.x to 2.1.0, configure the following items:
```
Comment (alejandrorascovan) [inline modules/ROOT/pages/upgrade.adoc]: 2.1.x

**Review by alejandrorascovan — CHANGES_REQUESTED** [review]

File: modules/ROOT/pages/backup-and-disaster-recovery.adoc (hunk 1 above)
Comment (paigehargrave) [reply modules/ROOT/pages/backup-and-disaster-recovery.adoc]: Got it - thank you!

File: modules/ROOT/pages/backup-and-disaster-recovery.adoc (hunk 2 above)
Comment (paigehargrave) [reply modules/ROOT/pages/backup-and-disaster-recovery.adoc]: Got it - thank you!

File: modules/ROOT/pages/backup-and-disaster-recovery.adoc (hunk 3 above)
Comment (paigehargrave) [reply modules/ROOT/pages/backup-and-disaster-recovery.adoc]: Removed - thank you!

File: modules/ROOT/pages/upgrade.adoc (hunk 5 above)
Comment (paigehargrave) [reply modules/ROOT/pages/upgrade.adoc]: Removed - thank you!

File: modules/ROOT/pages/upgrade.adoc (hunk 4 above)
Comment (paigehargrave) [reply modules/ROOT/pages/upgrade.adoc:7]: Added - thank you!

File: modules/ROOT/pages/upgrade.adoc (hunk 6 above)
Comment (paigehargrave) [reply modules/ROOT/pages/upgrade.adoc]: I removed this line - thank you!

File: modules/ROOT/pages/upgrade.adoc (hunk 7 above)
Comment (paigehargrave) [reply modules/ROOT/pages/upgrade.adoc]: Got it - thanks!

**Review by alejandrorascovan — APPROVED** [review]
LGTM

File: modules/ROOT/pages/backup-and-disaster-recovery.adoc [hunk 8]
```diff
@@ -3,7 +3,7 @@ ifndef::env-site,env-github[]
 include::_attributes.adoc[]
```
Comment (kbostrom) [inline modules/ROOT/pages/backup-and-disaster-recovery.adoc:3]: Title: please use only the approved abbreviation:

= Configure Backup and Restore for Anypoint Private Cloud  >
     = Configure Backup and Restore for Anypoint Platform PCE

File: modules/ROOT/pages/backup-and-disaster-recovery.adoc [hunk 9]
```diff
@@ -3,7 +3,7 @@ ifndef::env-site,env-github[]
...
 endif::[]
 
-To preserve your data in case of system failure, ensure that your system is automatically backed up. Configure your backup to run at least once per day; hourly is recommended. Store your backup archive on external storage outside the Anypoint cluster.
+To preserve your data in case of system failure or problems with other operations, such as patching or performing upgrades, ensure that your system is automatically backed up. Regular backups are extremely important for disaster recovery procedures. Configure your backup to run at least once per day; hourly is recommended. Store your backup archive on external storage outside the Anypoint cluster.
```
Comment (kbostrom) [inline modules/ROOT/pages/backup-and-disaster-recovery.adoc]: Anypoint Platform cluster:
```suggestion
To preserve your data in case of system failure or problems with other operations, such as patching or performing upgrades, ensure that your system is automatically backed up. Regular backups are extremely important for disaster recovery procedures. Configure your backup to run at least once per day; hourly is recommended. Store your backup archive on external storage outside the Anypoint Platform cluster.
```

File: modules/ROOT/pages/backup-and-disaster-recovery.adoc [hunk 10]
```diff
@@ -3,7 +3,7 @@ ifndef::env-site,env-github[]
...
 
 Anypoint Platform Private Cloud Edition (Anypoint Platform PCE) requires you to install, configure, and manage an NFS server. Ensure that you configure and perform frequent backups of this server.
```
Comment (kbostrom) [inline modules/ROOT/pages/backup-and-disaster-recovery.adoc:8]: Did you want to merge this into the previous statement, to fully introduce the product name and not repeat the backup recommendations?

File: modules/ROOT/pages/backup-and-disaster-recovery.adoc [hunk 11]
```diff
@@ -27,6 +27,22 @@ Specifies the deadline for the backup job, for example, 30s or 5m. Default is 20
...
+| Databases | Y
+| Configurations | Y
+| Secrets | Y
+| Object Store | Y
+| Drives | N
+| NFS files | N
+| Runtimes* | N
+|===
+ 
+*PCE only has INFORMATION about runtimes, such as their names and status.
```
Comment (kbostrom) [inline modules/ROOT/pages/backup-and-disaster-recovery.adoc:44]: This is such a small amount of data, even for a 2-col table.
I wonder if it would not be just as useful to make this narrative:

"The Anypoint Platform PCE backup operation backs up databases, configurations, secrets, and object stores. It does not back up drives, NFS files, or runtimes, although it does back up information about the runtimes, such as their names and statuses."

File: modules/ROOT/pages/backup-and-disaster-recovery.adoc [hunk 12]
```diff
@@ -27,6 +27,22 @@ Specifies the deadline for the backup job, for example, 30s or 5m. Default is 20
...
+
 == Perform a System Restore
 
 To restore a system from your backup, use the existing cluster or create a new cluster containing the same number of servers as the cluster being restored.
```
Comment (kbostrom) [inline modules/ROOT/pages/backup-and-disaster-recovery.adoc:48]: Is creating a new cluster actually considered a restore? I wonder if we should reword this to be something like "You can restore a system from you backup by using the existing cluster. Alternatively, you can create a new cluster containing..., provided that the versions of Anypoint Platform PCE match."

Comment (kbostrom) [reply modules/ROOT/pages/backup-and-disaster-recovery.adoc:48]: Line 55: consider...:

". Move the compressed backup archive file to a folder on any node in the environment to be restored securely, using the following command:"
+

File: modules/ROOT/pages/backup-and-disaster-recovery.adoc [hunk 13]
```diff
@@ -47,14 +63,20 @@ scp /backup-path/to-restore.tar.gz your_username@remotehost.edu:/var/lib/gravity
 ----
 sudo gravity restore /var/lib/gravity/planet/share/to-restore.tar.gz
 ----
-
++
 You can pass the following optional parameters:
-
++
+====
 * `--follow`: outputs the backup job logs to stdout
```
Comment (kbostrom) [inline modules/ROOT/pages/backup-and-disaster-recovery.adoc]: I would make each of these two lines instead, per our list style:
```suggestion
* `--follow`
Outputs the backup job logs to stdout
```

File: modules/ROOT/pages/backup-and-disaster-recovery.adoc [hunk 14]
```diff
@@ -47,14 +63,20 @@ scp /backup-path/to-restore.tar.gz your_username@remotehost.edu:/var/lib/gravity
...
 * `--timeout`: specifies the deadline for the backup job (e.g.: 30s, 5m, etc). Default is 20m.
 * `--log-file`: outputs logs to a file
+====
++
+Wait until the operation completes. This operation takes a few minutes.
```
Comment (kbostrom) [inline modules/ROOT/pages/backup-and-disaster-recovery.adoc]: Make this a step, because "waits" is an action...of sorts....
```suggestion
. Wait a few minutes for the operation to complete.
```
Note that "complete" is a transitive verb, which requires an object. See the style ref. for more about this, if you are interested.

File: modules/ROOT/pages/backup-and-disaster-recovery.adoc [hunk 15]
```diff
@@ -47,14 +63,20 @@ scp /backup-path/to-restore.tar.gz your_username@remotehost.edu:/var/lib/gravity
...
 * `--timeout`: specifies the deadline for the backup job (e.g.: 30s, 5m, etc). Default is 20m.
 * `--log-file`: outputs logs to a file
+====
++
+Wait until the operation completes. This operation takes a few minutes. 
 
-Note that this operation takes a few minutes. 
+. Manually restore the following information: 
++
+* NFS files
```
Comment (kbostrom) [inline modules/ROOT/pages/backup-and-disaster-recovery.adoc]: Wait...a list for just one thing?

File: modules/ROOT/pages/upgrade.adoc [hunk 16]
```diff
@@ -4,24 +4,28 @@ include::_attributes.adoc[]
 endif::[]
 
 == Before You Begin
```
Comment (kbostrom) [inline modules/ROOT/pages/upgrade.adoc:6]: Is there no text that introduces this task and under what circumstances the user wants or needs to perform it?
It that is the case, consider using the current Before You Begin (Prerequisites) text as that intro. 
It is odd to go from a Level-1 title straight to a Level-2.

File: modules/ROOT/pages/upgrade.adoc (hunk 4 above)
Comment (kbostrom) [inline modules/ROOT/pages/upgrade.adoc:7]: Making this just a little stronger:
```suggestion
Because Anypoint Platform Private Cloud Edition (Anypoint Platform PCE) upgrades require a working installation, all environment prerequisites must be already met. You cannot successfully upgrade your environment until you or your infrastructure team successfully performs all prerequisite tasks listed in xref:prereq-hardware.adoc[Hardware Prerequisites], xref:prereq-software.adoc[Software Prerequisites], and xref:prereq-network.adoc[Network Prerequisites] prior to the installation of Anypoint Platform PCE.
```

File: modules/ROOT/pages/upgrade.adoc [hunk 17]
```diff
@@ -4,24 +4,28 @@ include::_attributes.adoc[]
...
+Because Anypoint Platform Private Cloud Edition (Anypoint Platform PCE) upgrades require a working installation, all environment prerequisites should already be met. As long as you (or your infrastructure team) successfully performed all prerequisite tasks listed in xref:prereq-hardware.adoc[Hardware Prerequisites], xref:prereq-software.adoc[Software Prerequisites], and xref:prereq-network.adoc[Network Prerequisites] prior to the installation of Anypoint Platform PCE, your environment can be upgraded successfully.
 
-Refer to the xref:release-notes/pce/anypoint-private-cloud-2.0.3-release-notes#upgrade-paths[Upgrade Paths) section in the Anypoint Platform Private Cloud Edition (Anypoint Platform PCE) release notes for information about upgrading Anypoint Platform PCE to the most recent version.
+[NOTE]
```
Comment (kbostrom) [inline modules/ROOT/pages/upgrade.adoc:9]: This is actually more important than a note, and would follow nicely the text above it.

File: modules/ROOT/pages/upgrade.adoc [hunk 18]
```diff
@@ -4,24 +4,28 @@ include::_attributes.adoc[]
...
+If you are upgrading from Anypoint Platform PCE1.7.x to 2.0.x, an NFS server is required. Refer to xref:verify-nfs.adoc[NFS Server Prerequisites].
+
+Refer to the xref:release-notes::pce/anypoint-private-cloud-2.0.3-release-notes#upgrade-paths[Upgrade Paths) section in the Anypoint Platform PCE release notes for additional information about upgrading Anypoint Platform PCE to the most recent version.
 
 See xref:1.6@upgrade.adoc[About Migrating Anypoint Platform Private Cloud] for information on upgrading to version 1.6.1. See xref:1.7@upgrade.adoc[About Migrating Anypoint Platform Private Cloud] for information on upgrading to version 1.7.1.
 
-[WARNING]
-====
-Before upgrading:
+Contact MuleSoft professional services prior to upgrading for assistance.
```
Comment (kbostrom) [inline modules/ROOT/pages/upgrade.adoc:16]: Hmm. There are so many details to the prerequisites to upgrading, and then we end with sending the audience to MuleSoft professional services, anyway. Do we really have to go through all these details, if we are only sending them to get help?
Also, if there are specific prerequisites for versions and platforms, we might actually need--gasp!--a table! (I know, I know...Kim has an aversion to tables...lol....) . :-)

File: modules/ROOT/pages/upgrade.adoc [hunk 19]
```diff
@@ -4,24 +4,28 @@ include::_attributes.adoc[]
...
 
-* Ensure your environment meets all requirements listed in the xref:install-checklist[Product Prerequisites].
-* Perform a backup of your system as described in xref:backup-and-disaster-recovery.adoc[Configure Backup and Restore]. If you are upgrading from an older version that requires multiple intermediate upgrades you must backup each intermediate version before proceeding to the next.
+== Procedure
 
-During the upgrade process, there will be some downtime as each node is updated. 
-====
+The following steps describe how to upgrade Anypoint Platform PCE and the Anypoint Platform PCE infrastructure.
```
Comment (kbostrom) [inline modules/ROOT/pages/upgrade.adoc:20]: This intro text really does not add anything important to what the audience understands about what is happening here. Consider moving the Note text to compose the intro:

"Upgrading Anypoint Platform PCE also upgrades its infrastructure; all Anypoint Platform components are updated. You should expect [how much?] downtime for each node as it is updated."

File: modules/ROOT/pages/upgrade.adoc [hunk 20]
```diff
@@ -4,24 +4,28 @@ include::_attributes.adoc[]
...
 
-The following steps describe how to migrate Anypoint Platform PCE and the Anypoint Platform PCE infrastructure. You must also upgrade your applications to work with the new version of Anypoint Platform PCE. Contact your MuleSoft customer success representative for more information.
+[NOTE]
+====
+* During an upgrade, all Anypoint Platform PCE components are updated.
+* The upgrade process causes downtime for each node as it is updated.
+====
 
-== Procedure
+. Perform a backup of your system as described in xref:backup-and-disaster-recovery.adoc[Configure Backup and Restore]. If you are upgrading from an older version that requires multiple intermediate upgrades, you must backup each intermediate version before proceeding to the next.
```
Comment (kbostrom) [inline modules/ROOT/pages/upgrade.adoc:28]: Note that this is really not part of the upgrade process; it is a prerequisite, and one that has pretty much been beaten to death already, yes?
Step 1 should be the very first step of the actual upgrade process, including navigation if necessary.
```suggestion
. Perform a backup of your system as described in xref:backup-and-disaster-recovery.adoc[Configure Backup and Restore]. 
If you are upgrading from an older version that requires multiple intermediate upgrades, you must back up each intermediate version before proceeding to the next.
```

File: modules/ROOT/pages/upgrade.adoc [hunk 21]
```diff
@@ -4,24 +4,28 @@ include::_attributes.adoc[]
...
 
 . Obtain a link to the installer from your customer success representative.
```
Comment (kbostrom) [inline modules/ROOT/pages/upgrade.adoc:30]: I would call this a prerequisite as well, because a user should not find themselves in a task and missing this information....

Comment (kbostrom) [reply modules/ROOT/pages/upgrade.adoc:30]: I have edit comments about the following hidden text, and probably this topic set as a whole, which I should probably rereview when you have more time.

File: modules/ROOT/pages/upgrade.adoc [hunk 22]
```diff
@@ -71,17 +75,17 @@ the cluster is updating in background
 $ sudo gravity enter
 $ kubectl get pods
 ----
-+
-* Login to the Ops Center at the following URL:
+
+. Log in to the Ops Center at the following URL:
```
Comment (kbostrom) [inline modules/ROOT/pages/upgrade.adoc:79]:
```suggestion
. Log in to the Ops Center at https://{platform-host}:9500.
```
Will your audience be more familiar with Ops Center or Operations Center or operations center?

File: modules/ROOT/pages/upgrade.adoc [hunk 23]
```diff
@@ -71,17 +75,17 @@ the cluster is updating in background
...
 +
 ----
 https://{platform-host}:9500
 ----
 +
 [WARNING]
-Before accessing Anypoint Platform PCE, ensure that you configure your NSF servers described in the following step. Failure to configure NFS before accessing the platform can result in data loss.
+If you are upgrading from Anypoint Platform PCE 1.7.x to 2.0.x, ensure that you configure your NSF servers described in the following step before accessing Anypoint Platform PCE. In this case, failure to configure NFS before accessing the platform can result in data loss.
```
Comment (kbostrom) [inline modules/ROOT/pages/upgrade.adoc:86]: This does not belong in the task steps. 
The previous warnings AND the instructions that follow make clear what the user needs to do. 
It is not doc's job to account for corner-case users who can't follow instructions, if that is what is going on here.
If so, I would consider pushing back a bit if your dev reviewers are pressuring you for these constant warnings and notes; it sounds like the product has some flaws or lack of intuitive UI that the documentation cannot be sacrificed in an effort to rectify.

File: modules/ROOT/pages/upgrade.adoc [hunk 24]
```diff
@@ -71,17 +75,17 @@ the cluster is updating in background
...
 
-. After upgrading Anypoint Platform PCE, configure it to use the required NFS server.
+. If you are upgrading from from Anypoint Platform PCE 1.7.x to 2.0.x, configure the required NFS server.
```
Comment (kbostrom) [inline modules/ROOT/pages/upgrade.adoc]: Shouldn't they have done this already? Isn't this part of the prerequisites? We should not be documenting this over and over again.
```suggestion
. If you are upgrading from from Anypoint Platform PCE 1.7.x to 2.0.x, configure the required NFS server:
```

File: modules/ROOT/pages/upgrade.adoc [hunk 25]
```diff
@@ -90,6 +94,13 @@ Before accessing Anypoint Platform PCE, ensure that you configure your NSF serve
 .. Select *Save*.
 .. Select *Test NFS Server* to test that your NFS server is configured correctly.
 
+. Unless you are updating Anypoint Platform PCE from 2.0.x to 2.1.x, configure the following items:
```
Comment (kbostrom) [inline modules/ROOT/pages/upgrade.adoc:97]: ...and what happens if you are upgrading from 2.0 to 2.1? Are we again referring to prereqs in such a way that interferes with the main upgrade task?
If this is really a problem, maybe we should consider SEPARATE upgrade tasks for each situation?

**Review by kbostrom — APPROVED** [review]
Here you go!
Sorry this took so long, but I am really struggling with the repeated warnings and caveats getting in the way of the task. 
If this is so complicated, depending on versions and environments, then perhaps we need separate upgrade topics, rather than complicated this one overmuch?
